runners.gemspec

```
@@ -39,6 +39,7 @@ Gem::Specification.new do |spec|
   spec.add_dependency "bugsnag", ">= 6.18"
   spec.add_dependency "git_diff_parser", ">= 3.2"
   spec.add_dependency "jsonseq", ">= 0.2"
+  spec.add_dependency "nokogiri", ">= 1.11.7"
```

Please remove the patch version for consistency:

```suggestion
  spec.add_dependency "nokogiri", ">= 1.11"
```

---

Fixed it on https://github.com/sider/runners/pull/2427/commits/4c886f0610a0846042ade322e587a6e9c189d0fb
Thanks